### PR TITLE
Fix paperless, python.pkgs.django_extensions

### PR DESCRIPTION
**Pull request by erikarvstedt**
`python.pkgs.django_extensions` is a dependency of `paperless` and `python3Packages.hyperkitty`, which is still broken for other reasons.

### CODE REVIEW

File: pkgs/development/python-modules/django-extensions/default.nix [hunk 1]
```diff
@@ -16,6 +16,13 @@ buildPythonPackage rec {
     sha256 = "0053yqq4vq3mwy7zkfs5vfm3g8j9sfy3vrc6xby83qlj9wz43ipi";
   };
 
+  patches = [
```
Comment (jonringer) [inline pkgs/development/python-modules/django-extensions/default.nix]: do you mind adding a comment so it's easy to review is this is still applicable in future updates
```suggestion
  patches = [
    # fixes issue with ...., should be able to be removed in next release
```

**Comment by erikarvstedt** [comment]
@GrahamcOfBorg test paperless

File: nixos/modules/services/misc/paperless.nix [hunk 2]
```diff
@@ -122,10 +122,12 @@ in
...
 
-    systemd.tmpfiles.rules = [
-      "d '${cfg.dataDir}' - ${cfg.user} ${cfg.user} - -"
+    systemd.tmpfiles.rules = let
+      group = config.users.users.${cfg.user}.group;
+    in [
+      "d '${cfg.dataDir}' - ${cfg.user} ${group} - -"
     ] ++ (optional cfg.consumptionDirIsPublic
-      "d '${cfg.consumptionDir}' 777 ${cfg.user} ${cfg.user} - -"
+      "d '${cfg.consumptionDir}' 777 ${cfg.user} ${group} - -"
```
Comment (alyssais) [inline nixos/modules/services/misc/paperless.nix]: If this directory is 777 anyway (which is an… interesting choice), why not just make it root:root so you don’t have to worry about users?

Comment (erikarvstedt) [reply nixos/modules/services/misc/paperless.nix]: Yeah, that's simpler and functionally equivalent.

**Review by alyssais — APPROVED** [review]
Please autosquash :)

**Pull request merged**